larray/core/session.py

```diff
@@ -292,6 +416,20 @@ def filter(self, pattern=None, kind=None):
         -------
         Session
             The filtered session.
+            
+        Examples
+        --------
+        >>> s = Session(test1=ndtest((3, 3)), test2=ndtest((6, 4)), zero1=zeros((2, 8)), axis1=Axis('a=a0..a2'))                      
```

Comment:
I wonder if using intermediary variables for the arrays wouldn't make things clearer for users? OTOH having an inline example once in a while is good to remind them it is possible. Maybe use variables here and add an inline example in the class doctest?